src/main/java/net/minecraftforge/oredict/ShapelessOreRecipe.java

```diff
@@ -19,25 +19,25 @@
 
 package net.minecraftforge.oredict;
 
-import java.util.Iterator;
+import java.util.List;
 import net.minecraft.block.Block;
+import net.minecraft.client.util.RecipeItemHelper;
```

Comment:
Are you sure this is available on the server side? You might remove the side annotation, I guess.

Reply:
The class is available on both server side and client side.
In my opinion, mcp might make something wrong with the package of the class.

Reply:
No I didn't. Mojang did by making it server side in the hotfix. I didn't feel like nuking all mods by renaming it on a small patch.

Reply:
Pfew! Then again it compiled and worked here :D